tools/python/clean_strings_txt.py

```diff
@@ -64,6 +65,11 @@ def grep_ios():
     logging.info("Found in iOS: {0}".format(len(ret)))
     ret.update(get_hardcoded())
 
+    # iOS code scans resources for all available app_tip_XX strings.
+    grep = "grep app_tip_ {0}/data/strings/strings.txt".format(OMIM_ROOT)
+    ret2 = exec_shell(grep)
+    ret.update(parenthesize(strings_from_grepped(ret2, IOS_APPTIPS_RE)))
```

Comment:
This looks strange.
The point of this script is to compare usages in the codebase to the strings.txt.
This hack essentially makes it compare strings.txt to strings.txt :)

Is there a better way to process this exception?

Reply:
Please offer one if you have it )

Reply:
Maybe a special "forced" tag would do the job, but probably it doesn't justify the effort..